tests/e2e/lib/blocks/mailchimp.js

```diff
@@ -1,17 +1,17 @@
 /**
  * Internal dependencies
  */
-import { waitAndClick, waitForSelector, clickAndWaitForNewPage } from '../page-helper';
+import { waitForSelector } from '../page-helper';
 import LoginPage from '../pages/wpcom/login';
 import ConnectionsPage from '../pages/wpcom/connections';
 import logger from '../logger';
+import { getNgrokSiteUrl } from '../utils-helper';
 
 export default class MailchimpBlock {
-	constructor( block, page ) {
+	constructor( blockId, page ) {
```

Comment:
I'm curious why you decided to change the constructor signature? It is not explained in the description.

Reply:
It just looked simpler to me. This was during my attempt to debug/fix failures, and it's quite possible that some changes don't look like game changers. 
My first idea was actually to create a single `Block` class for all block (Mailchimp, Eventbride, etc.), that should get as a param the id or somethign to distinguish them. But then I saw there are more differences between the blocks and gave up. Still I think a parent block class with common functionality inherited by all blocks is a good idea.

Reply:
Yeah, that's all makes sense. And it makes total sense since we moving away from Gutenberg's e2e utils package. So we not receiving the full `blockInfo` anymore.